Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource/helper.go

```diff
@@ -117,7 +117,7 @@ func (m *Helper) Create(namespace string, modify bool, data []byte) (runtime.Obj
 			if err := m.Versioner.SetResourceVersion(obj, ""); err != nil {
 				return nil, err
 			}
-			newData, err := m.Codec.Encode(obj)
+			newData, err := m.RESTClient.GetCodec().Encode(obj)
```

Comment:
> I don't like the sig func (m *Helper) Create(namespace string, modify bool, data []byte) either, but changing it to func (m *Helper) Create(namespace string, modify bool, data interface{}) is more that I'd really like to change here

Especially since someone (@smarterclayton) thought about this upstream and decided that passing through straight `[]byte` was the proper way to build the API.  They went to some effort to make that happen.